core/src/common/lwm2m_plaintext.c

```diff
@@ -291,10 +291,10 @@ static int PTDeserialiseResource(SerdesContext * serdesContext, Lwm2mTreeNode **
 // Map PlainText serdes function delegates
 const SerialiserDeserialiser plainTextSerDes =
 {
-    .SerialiseObject   = NULL,
-    .SerialiseObjectInstance = NULL,
-    .SerialiseResource = PTSerialiseResource,
-    .DeserialiseObject = NULL,
-    .DeserialiseObjectInstance = NULL,
-    .DeserialiseResource = PTDeserialiseResource,
+    NULL,
```

Comment:
Personally, I'd prefer to use the named fields .. it's safer that way. But, it does depend upon having a compiler that understands that C dialect.